services/movie.py

```diff
@@ -0,0 +1,32 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    set_ = Movie.objects.all()
+    if genres_ids:
+        set_ = set_.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        set_ = set_.filter(actors__id__in=actors_ids)
+    return set_
+
+
+def get_movie_by_id(movie_id: int = None) -> Movie:
+    if movie_id:
+        return Movie.objects.get(id=movie_id)
```

Comment:
You don't need to use default value here. U should expect to retrieve proper `id` here